dll/win32/kernel32/winnls/string/lang.c

```diff
@@ -2366,24 +2366,24 @@ INT WINAPI CompareStringA(LCID lcid, DWORD flags,
     return ret;
 }
 
-static HANDLE NLS_RegOpenKey(HANDLE hRootKey, LPCWSTR szKeyName)
+HANDLE NLS_RegOpenKey(HANDLE hRootKey, LPCWSTR szKeyName)
```

Comment:
This file is (at least partially) Wine-synced.

Reply:
OK.